src/Components/CreateImageWizardV2/CreateImageWizard.tsx

```diff
@@ -197,7 +198,9 @@ const CreateImageWizard = () => {
             name="Review"
             id="step-review"
             footer={<CustomWizardFooter disableNext={true} />}
```

Comment:
The "Next" button should be "Create image" for this step, but I'm not sure if that would in the scope of the Review step PR as I can see it being a part of a PR that will implement the form submit functionality :thinking:

Reply:
I will wait to to submit funcionallty